package.json

```diff
@@ -9,10 +9,12 @@
     "test": "cross-env NODE_ENV=test npm -w packages/backend run test && npm -w packages/frontend run test",
     "generate-ssl-certs": "chmod +x ./scripts/generate-ssl-certs.sh && ./scripts/generate-ssl-certs.sh",
     "docker:dev": "docker compose -f ./docker/dev/docker-compose.yml --env-file .env.development up --build",
+    "docker:dev:fast": "docker compose -f ./docker/dev/docker-compose.yml --env-file .env.development up",
     "docker:dev:offline": "cross-env DOCKER_OFFLINE_MODE=true docker compose -f ./docker/dev/docker-compose.yml --env-file .env.development up",
     "docker:dev:prepare-offline": "docker compose -f ./docker/dev/docker-compose.yml --env-file .env.development build && docker compose -f ./docker/dev/docker-compose.yml --env-file .env.development pull",
     "docker:dev:frontend": "docker compose -f ./docker/dev/docker-compose.frontend-only.yml --env-file .env.development up --build",
     "docker:dev:down": "docker compose -f ./docker/dev/docker-compose.yml --env-file .env.development down",
+    "docker:dev:restart": "docker compose -f ./docker/dev/docker-compose.yml --env-file .env.development restart",
```

Comment:
Not sure what are the use-cases of this one? I cannot imagine a use-case where it will be helpful, it won't pick-up new env vars, hot-reload for modules is already implemented via mounted volumes, there's no way services will crash when developing locally and even if so it's just easier to hit Ctrl+C + `docker:dev`